export const getUserData=async()=>{
try{

}
catch(err){

}
}